force-app/main/default/classes/Homework.cls

```diff
@@ -1,60 +1,79 @@
 public with sharing class Homework {
     // Declare a variable to hold the postcode of Ginetta House - as
     // the office cannot move, this should be immutable
+    static final String postCode = 'WS13 8XG';
 
 
     // Declare a string that is initialised from an Integer
-
+    Integer numberOfStudents = 20;
+    String stringValueOfnumberOfStudents = String.valueOf(numberOfStudents);
 
     public void newYearsEve()
     {
         // Create a variable representing the time 12:30:25 in the local 
         // timezone and use this time to create a DateTime for the date 
         // 12/31/2021
+
+        Time localTime = Time.newInstance(12, 30, 25, 0);
+        // Datetime myNewYearEveDateTime = Date.NewInstance(2021, 12, 31) + Time.newInstance(12, 30, 25, 0);
     }
 
     public void castAccount()
     {
         // Create an sobject variable initialised to a new Account. 
         // Create an account variable that is initialised by casting the 
         // sobject variable.
+        
+
+        SObject mySObject = new Account (Name ='Brightgen');
+        Account acc = (Account) mySObject;
+        
     }    
 
     public void createBlob()
     {
         // Create a Blob from the characters ‘I am a Blob’
+        Blob myBlob = Blob.valueOf('I am a Blob');
     }
 
     public void createId()
     {
         // Create an ID with the value ‘00D180000001s0LEAQ', convert 
         // this to the 15 character version.
+        String newId = String.valueOf('00D180000001s0LEAQ');
+        Id myId = newId.substring(0, 15);
```

Comment:
While this works, it isn't future proofed if Salesforce change the way that 18 character Ids are converted to 15 character Ids. Since the Spring 21 release, a better method to use is : `to15()`